contracts/ssnlist.scilla

```diff
@@ -758,7 +758,13 @@ procedure WithdrawalStakeAmt(initiator: ByStr20, ssn: ByStr20, withdraw_amount:
   end
 end
 
-procedure CalcStakeDelegPerCycle(tmp_arg: TmpArg)
+procedure IncreaseReward(ssn: ByStr20, deleg: ByStr20, reward: Uint128)
```

Comment:
Why does it have the param `sssn` when it is not used in the transition?

Reply:
opps, you are correct.

Reply:
updated!